dictionaries/people-names/README.md

```diff
@@ -0,0 +1,89 @@
+# CSpell People Names Dictionary
+
+People Names dictionary for cspell.
+
+This is a pre-built dictionary for use with CSpell.
+
+This dictionary is an attempt at a collection of people names. It is not definitive. Contributions that add names are welcome.
+
+## Guidelines
+
+Due to the sensitive nature of names it is important to be clear about what is included in
+this dictionary. The idea here is to include names from all kinds of backgrounds and cultures, not just English or European (I call this out because this document was originally written in English including the examples below).
+
+- Each entry is part of a person's name, not the entire time.
+  For example, `John Smith` would not appear in the list, but `John` and `Smith` are ok.
+- No distinction is made between first, middle, last names. Each one can be an entry in the list.
+- All cultural and gender variants of a name are allowed.
+- A name should be common enough that 1000s of people have it.
```

Comment:
How can one verify the requirement?
Also I read it as "thousands" -- should it be more specific?

Reply:
Same remark

I think a google search or GitHub validation would be a good thing

Reply:
I'm trying to avoid made up names.

I like the Google search idea: "It is expected that a Google search of the name should return results confirming that it is a name."